05/app.js

```diff
@@ -1,3 +1,37 @@
-console.log('DOM');
+console.log('DOM')
 
-const curr = document.querySelector('.js-curr');
+const curr = document.querySelector('.js-curr')
+
+if (curr) {
+	const cloneBtn = curr.cloneNode(false)
+	cloneBtn.textContent = 'Usuń z koszyka'
+	curr.parentElement.appendChild(cloneBtn)
+}
+
+const currSiblings = curr.parentElement.children
+for (let i = 0; i < currSiblings.length; i++) {
+	const el = currSiblings[i]
+	el.classList.add('siblings')
```

Comment:
Należałoby wykluczyć element `.curr` :)

Reply:
Wygląda na to, że w tym przypadku powinienem wykluczyć cały kod? 🤔

Reply:
Nie, wystarczy, że zrobisz `if(curr !== el) { el.classList.add('siblings') }`